platforms/pictique/client/src/lib/fragments/Header/Header.svelte

```
@@ -110,6 +112,19 @@
 			</h1>
 		{/if}
 	</span>
+	{#if !route.includes('/settings')}
+		<button
+			type="button"
+			class={cn([
+				'cursor-pointer rounded-full p-2 hover:bg-gray-100 md:hidden',
+				classes.background
+			])}
+			onclick={() => goto('/settings')}
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)